runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -266,6 +266,10 @@ public class LootTrackerPlugin extends Plugin
 
 	private static final String WINTERTODT_SUPPLY_CRATE_EVENT = "Supply crate (Wintertodt)";
 
+	private static final String BAG_FULL_OF_GEMS_EVENT = "Bag full of gems (Percy)";
```

Comment:
these should be named eg `BAG_FULL_OF_GEMS_EVENT_PERCY`, `BAG_FULL_OF_GEMS_EVENT_BELONA` etc. Don't hard code ids into names.

Reply:
Would `BAG_FULL_OF_GEMS_PERCY_EVENT` be preferred over `BAG_FULL_OF_GEMS_EVENT_PERCY` to adhere to convention?

Reply:
either is fine

Reply:
Fixed with most recent commits 😁